provisioning/service/samples/createdelete_sample.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var assert = require('chai').assert;
```

Comment:
>assert [](start = 4, length = 6)

really awkward having a test dependency here and asserting in the code. not sure it's what we want our users to start doing.

Reply:
Removed.

---
In reply to: [147064651](https://github.com/Azure/azure-iot-sdk-node/pull/121#discussion_r147064651) [](ancestors = 147064651)